libfreerdp/core/gateway/wst.c

```diff
@@ -351,18 +381,60 @@ BOOL wst_connect(rdpWst* wst, DWORD timeout)
 
 	switch (StatusCode)
 	{
-		case HTTP_STATUS_NOT_FOUND:
-		case HTTP_STATUS_BAD_METHOD:
+		case HTTP_STATUS_FORBIDDEN:
```

Comment:
Not your code, but IIRC we had questions about "302 Moved" response...

Reply:
really?
If that's really needed I would suggest making a different PR. At least AVD seems to not need that